docs/database/database-spans.md

```diff
@@ -62,14 +62,16 @@ If there is no (low-cardinality) `db.operation.name` available, database span na
 SHOULD be [`{target}`](#target-placeholder).
 <!-- markdown-link-check-enable -->
 
+If neither `db.operation.name` nor `{target}` are available, span name SHOULD be `db.system`.
+
 Semantic conventions for individual database systems MAY specify different span name format.
 
-The <span id="target-placeholder">`{target}`</span> SHOULD adhere to one of the following values, arranged in prioritized order, provided they are accessible:
+The <span id="target-placeholder">`{target}`</span> SHOULD describe the entity operation is performed against
+and SHOULD adhere to one of the following values, provided they are accessible:
 
-- `db.collection.name`
-- `db.namespace`
-- `server.address:server.port`
-- `db.system`
+- `db.collection.name` SHOULD be used for data manipulation operations or operations on database collections.
+- `db.namespace` SHOULD be used only for operations on a specific database namespace.
+- `server.address:server.port` SHOULD be used for other operations not targeting a specific database or collection
```

Comment:
(trying to avoid instrumentation authors thinking that they should use this if operations target multiple databases or multiple collections as opposed to a single specific one)

```suggestion
- `server.address:server.port` SHOULD be used for other operations not targeting any specific database(s) or collection(s)
```